### PR TITLE
refactor: add shortcut onboarding slide to onboarding carousel

### PR DESCRIPTION
**Pull request by patricio0312rev**
# [[onboarding] add shortcut slide](https://app.clickup.com/t/86du13mn7)

## Summary

- Icons for shortcut slide have been added.
- `Progressbar` component has been refactor to support `n`slides.
- New `ShortcutIcon` component.
- The slide with shortcut info has been added to the onboarding carousel.

image

## Checklist

- [ ] I checked that both `pnpm dev` and `pnpm dev:bare` work as intended
- [ ] I checked the basic extension interactions and made sure wallet selection works
- [ ] I checked my UI changes against the design and there are no notable differences, including responsiveness
- [ ] I checked my (code) changes for obvious issues, debug statements and commented code
- [ ] I opened a corresponding card on Clickup for any remaining TODOs in my code
- [ ] I added a short description on how to test this PR _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

**Review by samharperpittam — APPROVED** [review]
Can't see the green variant atm but the blue looks fine. Will take a look at green once a fix is in place for it.

**Comment by ItsANameToo** [comment]
> Can't see the green variant atm but the blue looks fine. Will take a look at green once a fix is in place for it.

go to `src/lib/store/ui/index.ts` and change the `initialState` from `ThemeAccent.NAVY` to `ThemeAccent.GREEN`

File: src/i18n/ns/pages.ts [hunk 1]
```diff
@@ -148,6 +148,7 @@ export default {
             CONTROL_YOUR_IDENTITY:
                 'Control your identity with our <br /> session management feature.',
             SIGN_TRANSACTIONS: 'Sign transactions and <br /> perform on-chain actions.',
+            SHORTCUT: 'You can open the extension by <br /> pressing Shift + Option + A.',
```
Comment (ItsANameToo) [inline src/i18n/ns/pages.ts:151]: this needs changing depending on OS

**Review by ItsANameToo — CHANGES_REQUESTED** [review]

File: src/i18n/ns/pages.ts (hunk 1 above)
Comment (patricio0312rev) [reply src/i18n/ns/pages.ts]: Fixed ✅

File: src/pages/Onboarding.tsx [hunk 2]
```diff
@@ -20,20 +22,21 @@ type OnboardingScreen = {
 
 const Onboarding = () => {
     const { t } = useTranslation();
-
-    const navigate = useNavigate();
-
-    const [activeOnboardingScreen, setActiveOnboardingScreen] = useState<number>(0);
+    const [os, setOs] = useState<string>('default');
```
Comment (patricio0312rev) [inline src/pages/Onboarding.tsx:25]: Since this is being used in 3 files now, I'll do a refactor for when everything gets merged.

**Review by samharperpittam — APPROVED** [review]

**Review by ItsANameToo — APPROVED** [review]

**Pull request merged**